apis/python/src/tiledbsoma/io/spatial/__init__.py

```diff
@@ -8,7 +8,20 @@
 may be dropped.
 """
 
-from .ingest import VisiumPaths, from_visium, register_visium_datasets
+from .ingest import (
+    VisiumPaths,
+    XeniumPaths,
+    from_visium,
+    from_xenium,
+    register_visium_datasets,
+)
 from .outgest import to_spatialdata
 
-__all__ = ["to_spatialdata", "from_visium", "register_visium_datasets", "VisiumPaths"]
+__all__ = [
+    "to_spatialdata",
+    "from_visium",
+    "register_visium_datasets",
+    "VisiumPaths",
+    "from_xenium",
+    "XeniumPaths",
+]
```

Comment:
Let's not add `from_xenium` and `XeniumPaths` to the module until it's fully implemented.